openedx/features/badging/helpers.py

```diff
@@ -71,6 +71,10 @@ def get_course_badges(user, course_id, earned_badges, badge_queryset=None):
                 # user has not joined any team
                 badges['team_joined'] = False
 
+            if course_team and 'team_id' in course_team and course_team['team_id']:
```

Comment:
Please verify, Do we really need  `and 'team_id' in course_team and course_team['team_id']` ? ,  I think we can remove this according to this query `course_team = CourseTeam.objects.filter(course_id=course.id, users=user).values('team_id', 'team__room_id').first()`

Reply:
Done